Makefile

```diff
@@ -64,10 +64,15 @@ build-mac-arm64:
 		-o bin/darwin/arm64/$(CLI_EXE) $(CLI_PKG)
 
 .PHONY: build-windows
-build-windows:
+build-windows: build-windows-amd64 build-windows-arm64
+.PHONY: build-windows-amd64
+build-windows-amd64:
```

Comment:
A empty line is required between makefile items for consistency and readability.

For example:
```
a:
    Build a

b:
    Build b

c:
    Build c
```

Reply:
I just checked if modified target can be made, it's better to add empty line between items.